parse_error_log_test: sirepo srw create-predefined needs own config

This is actually a broader bug in that each test relies on resources being created by previous tests. Even broader, predefined.json should not be created statically. 

parse_error_log_test is trying to run in non-dev mode. This is actually pretty hard to do, and in this case, this condition should not be on in_dev_mode, but instead on a config that defaults in dev mode. 

It works in the test suite, because parse_error_log_test is not the first test.

```txt
  File "/home/vagrant/src/radiasoft/sirepo/tests/conftest.py", line 39, in fc_module
    with srunit_servers.api_and_supervisor(request, fc_args=a) as c:
  File "/home/vagrant/.pyenv/versions/3.9.15/lib/python3.9/contextlib.py", line 119, in __enter__
    return next(self.gen)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/srunit_servers.py", line 147, in api_and_supervisor
    template.import_module("srw").get_predefined_beams()
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/srw.py", line 452, in get_predefined_beams
    return _SIM_DATA.srw_predefined().beams
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/sim_data/srw.py", line 354, in srw_predefined
    assert (
AssertionError: predefined.json: not found; call "sirepo srw create-predefined" before pip install
```

--- a/tests/parse_error_log_test.py
+++ b/tests/parse_error_log_test.py
@@ -14,6 +14,8 @@ def setup_module(module):
         # Need to set these manually when in non-dev-mode
         SIREPO_COOKIE_IS_SECURE="0",
         SIREPO_COOKIE_PRIVATE_KEY="MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTI=",
+        # Avoids srw/predefined.json check
+        SIREPO_FEATURE_CONFIG_SIM_TYPES="myapp",
         SIREPO_SMTP_PASSWORD="dev",
         SIREPO_SMTP_SERVER="dev",
         SIREPO_SMTP_USER="dev",

